openedx/core/djangoapps/courseware_api/utils.py

```diff
@@ -29,15 +29,15 @@ def get_celebrations_dict(user, enrollment, course, browser_timezone):
     celebrations = {
         'first_section': CourseEnrollmentCelebration.should_celebrate_first_section(enrollment),
         'streak_length_to_celebrate': streak_length_to_celebrate,
-        'streak_discount_experiment_enabled': False,
+        'streak_discount_enabled': False,
     }
 
-    # We only want to bucket people into the AA-759 experiment if they are going to see the streak celebration
+    # We only want to bucket people into the streak discount if they are going to see the streak celebration
```

Comment:
the "bucket" text is relevant to an experiment with two buckets, so that can be changed